src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ZombieClusterMonitorSpec.scala

```diff
@@ -131,7 +131,7 @@ class ZombieClusterMonitorSpec extends TestKit(ActorSystem("leonardotest")) with
     }
   }
 
-  it should "should not zombify Creating clusters" in isolatedDbTest {
+  it should "should zombify creating cluster after hang period" in isolatedDbTest {
```

Comment:
Do we need to keep the original test that will not zombify creating cluster if it's within hang period?

Reply:
I was thinking the same thing on the first pass. Just added it and cleaned up the hardcoded constants